tests/integration/synapseclient/integration_test.py

```diff
@@ -870,3 +865,269 @@ def test_get_permissions_for_project_with_public_and_registered_user(self):
             "DOWNLOAD",
         ]
         assert set(expected_permissions) == set(permissions)
+
+
+class TestPermissionsOnEntityForCaller:
+    @pytest.fixture(autouse=True, scope="function")
+    def init(self, syn: Synapse, schedule_for_cleanup):
```

Comment:
SonarCloud is flagging these `schedule_for_cleanup` parameters for not having a function parameter. We should be able to use:
```
from typing import Callable
```

```suggestion
    def init(self, syn: Synapse, schedule_for_cleanup: Callable[..., None]):
```

In this case.